spec/controllers/cafe_works_controller_spec.rb

```diff
@@ -68,14 +68,16 @@
         end
 
         it 'update worker' do
-          patch :update_worker, {id: cwork_profile.to_param, cafe_work: attributes_for(:assignee, :test)}
+          patch :update_worker,
+            id: cwork_profile.to_param,
```

Comment:
@henrikssn 
What should we do about these ones?
I either need to change my Vim-config or we should edit Hound.